packages/holocron/src/createHolocronStore.js

```diff
@@ -85,41 +87,40 @@ function createReducer({ appReducer, state, localsForBuildInitialState }) {
   };
 }
 
-const holocronEnhancer = (localsForBuildInitialState, extraThunkArguments = {}) =>
-  createReduxStore => (appReducer, preloadedState, enhancer) => {
-    const reducer = createReducer({
-      appReducer,
-      state: preloadedState,
-      localsForBuildInitialState,
-    });
-    const store = createReduxStore(reducer, preloadedState, enhancer);
-    const { getState } = store;
-    const rebuildReducer = () =>
-      store.replaceReducer(createReducer({
-        appReducer,
-        state: getState(),
-        localsForBuildInitialState,
-      }));
-    store.rebuildReducer = rebuildReducer;
-    let { dispatch } = store;
-
-    if (!global.BROWSER) { store.modules = getModules(); }
-
-    const middlewareAPI = {
-      getState: store.getState,
-      dispatch: action => dispatch(action),
-    };
-    dispatch = thunk.withExtraArgument({
-      ...extraThunkArguments,
-      rebuildReducer,
-      modules: store.modules,
-    })(middlewareAPI)(store.dispatch);
-
-    return {
-      ...store,
-      dispatch,
-    };
+const holocronEnhancer = (localsForBuildInitialState, extraThunkArguments = {}) => (
```

Comment:
if you have the arguments on a new line after `(` then the closing `)` should also be on a new line to help readability.

```js
const myFunction = (
  arg1,
  arg2
) => (
  // some code
);
```
or you can keep them on the same line.
```js
const myFunction = (arg1, arg2) => (
  // some code
);
```

Reply:
Thank you Johnny i've updated this on 24de93f